### ISSUE TITLE
Support for Universal Windows Platform (Windows 10)

### ISSUE THREAD

**Issue by azunyuuuuuuu** [canton7/Stylet#8]
Hello,
are there any plans to support the Universal Windows Platform?
Or is eventually a port planned?

**Comment by canton7** [comment]
I've honestly never considered: I don't develop for UWP myself.

I'll have a look and see how much effort it would be to adapt Stylet for UWP. It will take a little bit of time: I do most of my open source dev over my lunch breaks and my work PC is Windows 7.

**Comment by 10Dev** [comment]
You might find the discussion here to be useful:

https://github.com/mbdavid/LiteDB/issues/86

This Microsoft utility is mentioned:

https://github.com/Microsoft/dotnet-apiport

**Comment by qq840570226** [comment]
will you transplante to .net core 3.0 and support Universal Windows Platform?
Since MS going to abandon .net Framework and end it in 4.8.

**Comment by canton7** [comment]
It's not like .NET Framework is going away - SyncTrayzor will keep working.

There may be a good reason to switch to .NET Core 3.0 in the future, but that will increase the download size by a *lot*, so I'm not too keen to do it. UWP will probably never be supported.

**Comment by sigmarsson** [comment]
I just tried to build a .NET 5 application and the **s:** namespace could not even be discovered, though the same xaml works properly in another 4.7.2 WPF application.

![image](https://user-images.githubusercontent.com/43340959/99521627-f2c7a180-2994-11eb-8556-0a84ab05e8b4.png)

Here is my .**csproj**:

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net5.0-windows10.0.18362.0</TargetFramework>
    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
    <RootNamespace>Weather.History.WinUI</RootNamespace>
    <ApplicationManifest>app.manifest</ApplicationManifest>
    <Platforms>x86;x64;</Platforms>
    <RuntimeIdentifiers>win10-x86;win10-x64;</RuntimeIdentifiers>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.VCRTForwarders.140" Version="1.0.7" />
    <PackageReference Include="Microsoft.WinUI" Version="3.0.0-preview3.201113.0" />
    <PackageReference Include="Stylet" Version="1.3.4" />
    <Manifest Include="$(ApplicationManifest)" />
  </ItemGroup>
</Project>
```

**Comment by sigmarsson** [comment]
Will you support **.NET 5** as **TargetFramework** ?

**Comment by canton7** [comment]
This issue is about UWP, not WPF on .NET 5.

The intention is to support .NET 5, multi-targetting if necessary. I'll try and find time to take a look over the weekend.

**Comment by canton7** [comment]
Turns out Stylet works just fine on .NET 5.

Note that your csproj doesn't have `<UseWpf>true</UseWpf>`, and instead looks like it's a WinUI app. Stylet is a WPF framework, and currently has no support for WinUI.

**Comment by azunyuuuuuuu** [comment]
> Turns out Stylet works just fine on .NET 5.
> 
> Note that your csproj doesn't have `<UseWpf>true</UseWpf>`, and instead looks like it's a WinUI app. Stylet is a WPF framework, and currently has no support for WinUI.

I did a small demo app at work a few weeks ago written in .net 5.0 and used Stylet and noticed that it just works out of the box. I did not even need to deviate from the manual to get it working.

I wonder how well UWP is going to be supported now that Windows Phone is dead, etc. 🤔. I do not think that it is still necessary to get Stylet to work with UWP. And I do think that this issue can be closed, except if someone still wants this.
